apps/webapp/src/components/header/sync-status-section.tsx

```diff
@@ -0,0 +1,24 @@
+import { useStream } from '../../fetchers/stream';
+import { useMemo } from 'react';
+import { viewClient } from '../../clients/grpc';
+import { BlockSyncStatus } from '@penumbra-zone/ui';
+
+export const SyncStatusSection = () => {
```

Comment:
Totally not necessary to change the UI for this PR, but my feeling is that the block sync takes up a lot of above-the-fold space for something that probably should be more back-of-mind for users. Instagram, for example, has a bar that goes across the _very_ top of the screen (which we could also augment with the words "Blocks synced ✔️"). Again, not necessary for this PR, but just thinking about a future improvement.



https://github.com/penumbra-zone/web/assets/1121544/1736c09e-adf4-4cdd-99fd-55f6b6838b30

Reply:
Ah, you are so right. Relocating to the top.